msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/nonjoined/TestCase1561158.java

```diff
@@ -1,60 +0,0 @@
-//  Copyright (c) Microsoft Corporation.
```

Comment:
why are we removing these tests as opposed to ignoring them? Are they impossible to fix?

Reply:
Most of these are now integration tests that pedro created. this one wasn't a valid scenario